at_client/lib/src/client/at_client_spec.dart

```diff
@@ -62,7 +62,7 @@ abstract class AtClient {
   ///    put(key, '+1 999 9999');
   /// ```
   /// Starting version 3.0.0 [isDedicated] is deprecated
-  Future<bool> put(AtKey key, dynamic value, {bool isDedicated = false});
+  Future<bool> put(AtKey key, dynamic value);
```

Comment:
why now removing rather than leaving in place? I thought we were keeping backwards compatibility in this change

Reply:
Added the isDedicated argument back to method signature.